### PR TITLE
WIP: Add attributes and controls to columns

### PR DESCRIPTION
**Pull request by carolinan**
Adds the following new attributes:

- Columns. The number of columns. In case we need to pass the max number of columns to the inner block settings using [ProvidesContext](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-context/). - _To do: If we want to use this, it needs to be assigned the value from `count`._ _Range control._
- layoutSwitch. Switch been display: flex (default) and display: grid. _Toggle._
- flexDirection. _Select._
- breakpointDirection. Wether to change flex direction on a specific breakpoint, and which direction. _Select._
- flexWrap.  _Toggle._

Adds placeholder classes depending on the layoutSwitch
Adds inline style depending on the layoutSwitch, direction and wrap.

I did not include the % and with controls from the core/columns block for now, because they are one of the problems with the core/columns block and I am not sure what solution for the width of the inner blocks we will use (for the flex layout).

### CODE REVIEW

File: columns/block.json [hunk 1]
```diff
@@ -7,5 +7,26 @@
...
+		},
+		"layoutSwitch": {
+			"type": "boolean",
+			"default": false
+		},
+		"flexDirection": {
+			"type": "string",
+			"default": "row"
+		},
+		"breakpointDirection": {
```
Comment (carolinan) [inline columns/block.json:22]: This needs a better name.

File: columns/src/edit.js [hunk 2]
```diff
@@ -1,16 +1,193 @@
...
+								] }
+								value={ flexDirection }
+								onChange={ ( value ) =>
+									setAttributes( { flexDirection: value } )
+								}
+							>
+							</SelectControl>
+							<SelectControl
+								label={ 'Breakpoint Flex Direction' }
+								options={ [
```
Comment (carolinan) [inline columns/src/edit.js:122]: These options are repeated and should be simplified and reused :)

**Comment by carolinan** [comment]
I have also not touched the front / the save function.

File: columns/src/edit.js (hunk 2 above)
Comment (aristath) [reply columns/src/edit.js]: 👍 Moved the options to a const, and rebased the PR

**Pull request merged**